SPECS/espeak-ng/espeak-ng.spec

```diff
@@ -1,16 +1,17 @@
 Summary:        Compact text-to-speech synthesizer
 Name:           espeak-ng
-Version:        1.50
-Release:        3%{?dist}
+Version:        1.51.1
+Release:        1%{?dist}
 # Apache2 license applies only to Android APK code- does not apply here
 # BSD license applies only to Windows code- does not apply here
-License:        GPLv3 AND Unicode
+License:        GPL-3.0-only AND GPL-3.0-or-later AND Unicode-DFS-2016 AND CC-BY-SA-3.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/espeak-ng/espeak-ng
-Source0:        https://github.com/%{name}/%{name}/releases/download/%{version}/%{name}-%{version}.tgz
-Patch0:         tests-fix-greek-letter-variants.patch
-Patch1:         tests-newline-fixes.patch
+Source0:        https://github.com/%{name}/%{name}/archive/%{version}/%{name}-%{version}.tar.gz#/%{name}-%{version}.tgz
+# Backported from:
+# https://github.com/espeak-ng/espeak-ng/commit/58f1e0b6a4e6aa55621c6f01118994d01fd6f68c
```

Comment:
If these are not needed, we can remove these lines.

Reply:
Thanks. Not needed will remove this.